src/content/1.7/modules/testing/resources.md

```diff
@@ -0,0 +1,148 @@
+
+This feature allows an automatic configuration of your module but DOES NOT display the configuration page.
+
+This feature targets any developers who would like to test its modules after an installation without having to pass the configuration screen, or an agency who has a generic configuration to set on every shops.
+With only one command, we expect a module to be totally functional for tests or production.
+
+If we reuse the previous command, we could expect something like this:
+```term
+php bin/console prestashop:module configure <module name> [<YML file path>]
+```
+
+{{% notice note %}}
+Use `php app/console` instead of `php bin/console` for versions prior {{< minver v="1.7.4" >}}
+{{% /notice %}}
+
+
+#### Source of data configuration
+When not provided, we use by default a file `self_config.yml` stored in the module root folder. You can use another file by setting the the `<YML file path>`.
+
+#### Format supported
+We are going to make it simple and allow only file extension: yaml.
+
+Example:
+
+```yml
+# This file is an example of data configuration which can be applied to a module
+
+# Data to be updated in Configuration table
+configuration:
+    update:
+        # Option 1: having a pair key/value
+        PAYPAL_SANDBOX: 1
+        PAYPAL_API_CARD: 0
+        # Option 2: use "value" subkey. Will allow to use additional keys later
+        PAYPAL_SANDBOX:
+            value: 1
+            multilang: false
+        # Question: how to handle multi lang values ?
+    delete:
+        - "PAYPAL_ONBOARDING"
+
+# SQL files to execute
+sql:
+    - file: "path/to/file1.sql"
+    # But key is optional
+    - "path/to/file2.sql"
+    - "sql/default-config.sql"
+
+# File to copy from local disk or website
+files:
+    - source: "/path/to/source/file.txt"
+      dest: "/module/path/docs/file.txt"
+
+    - source: "https://www.domain.wow/uri/of/file"
+      dest: "/module/path/docs/another_file.txt"
+
+# Fox complex configuration data, allow PHP code to be executed
+php:
+    - file: "path/to/file.php"
+       # Must have a class which implements interface PrestaShop\PrestaShop\Adapter\Module\Configuration\ModuleComplexConfigurationInterface
+      params:
+        - myParam1: 1
+        - oneArrayParam:
+            - "value1"
+            - "value2"
+            - withSpecificKey: "value3"
+```
+
+
+## Core Functional tests
+
+PrestaShop provides its own test suite, running with selenium. It covers the features of PrestaShop and grows each time a bug is resolved or a feature is added.
+
+These tests are launched every time a change is suggested to the core, but you can also run them with your module installed. You will certify it did not break a critical feature of the core.
+
+This section will be completed when these tests will be available on a dedicated repository. In the meantime you can already reach them on GitHub, in the [tests/E2E folder](https://github.com/PrestaShop/PrestaShop/tree/1.7.3.x/tests/E2E) of PrestaShop files.
```

Comment:
@Quetzacoalt91 link should point to latest `develop` tests

Reply:
Indeed the folder is obsolete in the latest versions of PrestaShop. However I'm not in favor of relying on the develop branch, because if the folder disappear on a future minor version, this would lead the visitor on a HTTP 404.
Note this happened to the previous link. The folder tests/E2E does not exist anymore on the latest branches.